packages/core/src/recon.ts

```diff
@@ -47,13 +49,41 @@ export class ReconApi implements IReconApi {
   }
 
   async init(): Promise<void> {
+    if (!this.enabled) {
+      return
+    }
+
     this.#url = await this.#config.url
+    await this.registerInterest(Model.MODEL)
+  }
+
+  async registerInterest(model: StreamID): Promise<void> {
+    if (!this.enabled) {
+      throw new Error(`Recon: disabled, not registering interest in model ${model.toString()}`)
+    }
+
+    try {
+      await this.#sendRequest(this.#url + `/ceramic/interests/model/${model.toString()}`, {
+        method: 'POST',
+      })
+      this.#logger.debug(`Recon: added interest for model ${model.toString()}`)
+    } catch (err) {
+      this.#logger.err(
+        `Recon: failed to register interest in model ${model.toString()} with error ${err}`
+      )
+      throw err
+    }
   }
 
   async put(event: ReconEvent, opts: AbortOptions): Promise<void> {
+    if (!this.enabled) {
+      this.#logger.imp(`Recon: disabled, not putting event ${event.id}`)
+      return
+    }
+
     const body = {
-      id: event.id.toString(),
-      data: event.data.toString(),
+      eventId: event.id.toString(),
+      eventData: event.data.toString(),
```

Comment:
I think the PR merged to change this back to `id` and `data`. @dav1do Can you confirm?

Reply:
dangit!!!!! i gotta flip that back

Reply:
They are currently both supported.. but I was planning to delete the old version once I sort out the e2e tests: https://github.com/ceramicnetwork/rust-ceramic/pull/272